src/Backend/SgateShopgatePlugin/Controllers/Frontend/shopgate.php

```diff
@@ -662,7 +662,8 @@ public function updateUserEmailAction()
         $this->Response()->setHeader('Content-Type', 'application/json');
         $this->Response()->setBody(json_encode($response));
         $this->Response()->sendResponse();
-        exit();
+        $this->Response()->send();
```

Comment:
Is this line needed here? I don't see it in the other methods, just wondering.

Reply:
No, it is not required. I overlooked it when refactoring - that's the Symfony method.

Reply:
```suggestion
```